kima/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "kima",
+  "assets": [
+    {
+      "description": "The native token of Kima Chain",
+      "denom_units": [
+        {
+          "denom": "uKIMA",
```

Comment:
Now that mainnet is ready, please mark the testnet asset as a "testnet-mintage" of this mainnet asset.

Reply:
From what I understand that is the addition of the following to our testnet config: 

```json
      "traces": [
        {
          "type": "test-mintage",
          "counterparty": {
            "chain_name": "kima",
            "base_denom": "uKIMA"
          },
          "provider": "Kima Network"
        }
      ],
```

> What exactly does this mean?

Reply:
Yes, that it correct.
I just helps show how test mintages correspond to mainnet counterparts, and allows the assets to share metadata like logo image, description, and socials.